bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/pom.xml

```diff
@@ -52,6 +55,18 @@ limitations under the License.
   the enduser can explcitly add their own version hbase-server.
   -->
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-bom</artifactId>
+        <version>${google-cloud-bom.version}</version>
+        <type>pom</type>
+        <scope>test</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
```

Comment:
This looks a bit off:
1. boms should be imported with scope import
2. we shouldnt be using the google-cloud-bom in hadoop jobs

Reply:
good catch. removed.